blossom/templates/app/landing_page.html

```diff
@@ -0,0 +1,9 @@
+{% extends "website/partials/base.partial" %}
+
+{% block content %}
+    {% if not request.user.accepted_coc %}
+    ...
+    {% else %}
```

Comment:
Is this still needed? If I remember correctly the CoC is already shown correctly